traits/trait_set_object.py

```diff
@@ -98,8 +98,6 @@ def __init__(self, trait, object, name, value):
 
             super(TraitSetObject, self).__init__(value)
 
-            return
```

Comment:
Much excitement: a case of a redundant return that _isn't_ at the end of a function or method!